packages/@n8n/nodes-langchain/nodes/vector_store/VectorStoreSupabase/VectorStoreSupabase.node.ts

```diff
@@ -86,17 +84,32 @@ export const VectorStoreSupabase = createVectorStoreNode({
 		});
 	},
 	async populateVectorStore(context, embeddings, documents, itemIndex) {
-		const tableName = context.getNodeParameter('tableName', itemIndex) as string;
+		const tableName = context.getNodeParameter('tableName', itemIndex, '', {
+			extractValue: true,
+		}) as string;
 		const options = context.getNodeParameter('options', itemIndex, {}) as {
 			queryName: string;
 		};
 		const credentials = await context.getCredentials('supabaseApi');
 		const client = createClient(credentials.host as string, credentials.serviceRole as string);
 
-		void SupabaseVectorStore.fromDocuments(documents, embeddings, {
-			client,
-			tableName,
-			queryName: options.queryName ?? 'match_documents',
-		});
+		try {
+			await SupabaseVectorStore.fromDocuments(documents, embeddings, {
+				client,
+				tableName,
+				queryName: options.queryName ?? 'match_documents',
+			});
+		} catch (error) {
+			if (error.message === 'Error inserting: undefined 404 Not Found') {
```

Comment:
Checking for a specific text of the error message seems fragile to me, as the exact wording could change. Can we check for the status code instead?

Reply:
unfortunately error does not contain any other identifiers